storage/database/src/table.rs

```diff
@@ -8,12 +8,7 @@ use crate::{key::Key, storable::Storable};
 /// Database table metadata.
 ///
 /// Purely compile time information for database tables.
-///
-/// ## Sealed
-/// This trait is [`Sealed`](https://rust-lang.github.io/api-guidelines/future-proofing.html#sealed-traits-protect-against-downstream-implementations-c-sealed).
-///
-/// It is only implemented on the types inside [`tables`][crate::tables].
-pub trait Table: crate::tables::private::Sealed + 'static {
+pub trait Table: 'static {
```

Comment:
## `cuprate_database::Table` 1/4
Since `cuprate_database` doesn't decide the tables anymore, this is no longer sealed. Any table can be created.